src/h5web/toolbar/controls/DomainSlider/BoundEditor.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactElement, useEffect, useRef, useState } from 'react';
+import { FiCheck, FiSlash } from 'react-icons/fi';
+import { formatValue } from '../../../utils';
+import styles from './BoundEditor.module.css';
+
+interface Props {
+  label: string;
+  value: number;
+  isEditing: boolean;
+  hasError: boolean;
+  onEditToggle: (force: boolean) => void;
+  onChange: (val: number) => void;
+}
+
+function BoundEditor(props: Props): ReactElement {
+  const { label, value, isEditing, hasError, onEditToggle, onChange } = props;
+
+  const id = `${label}-bound`;
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const [inputValue, setInputValue] = useState(formatValue(value));
+
+  useEffect(() => {
+    setInputValue(isEditing ? value.toString() : formatValue(value));
```

Comment:
I would find more intuitive if the value was staying under `.3e` format when editing.

But that's my personal opinion so let's wait for user feedback before changing.

Reply:
Ah yeah? It would allow me to shrink back the size of the input fields and to not have to reduce the font-size in editing mode. It wouldn't prevent people from entering values with as many decimals as they want, so... why not!

Reply:
> so let's wait for user feedback before changing.

yeah agreed